front/src/modules/ui/board/hooks/useCurrentCardSelected.ts

```diff
@@ -13,19 +14,48 @@ export function useCurrentCardSelected() {
     isCardSelectedFamilyState(currentCardId ?? ''),
   );
 
+  const [_, setActiveCardIds] = useRecoilState(activeCardIdsState);
```

Comment:
You can use useSetRecoilState : https://recoiljs.org/docs/api-reference/core/useSetRecoilState/

Reply:
oh didn't know that existed! So cool